src/Orc.NuGetExplorer.Xaml/Commands/PackagesBatchUpdateCommandContainer.cs

```diff
@@ -28,13 +27,6 @@ internal class PackagesBatchUpdateCommandContainer : CommandContainerBase<IManag
             IExtensibleProjectLocator projectLocator, IPackageCommandService packageCommandService, IPackageOperationContextService packageOperationContextService)
             : base(Commands.Packages.BatchUpdate, commandManager)
         {
-            Argument.IsNotNull(() => progressManager);
```

Comment:
Don't remove the argument checks. Replace them with calls to ArgumentNullException.ThrowIfNull(progressManager)